### PR TITLE
Changed occurences of "data table" to "table" to reflect the componen…

### PR DESCRIPTION
**Pull request by ncameronbritt**
…t name.

Partially addresses https://github.com/patternfly/patternfly-next/issues/1993

### CODE REVIEW

**Comment by LHinson** [comment]
@mcarrano want to take a look at this one as well? 

@ncameronbritt looks like you got conflicts :/ Do you use sourcetree to send PRs? If so, I can help you. If not, let me know what you use and we'll get someone else to help you!

**Comment by ncameronbritt** [comment]
@LHinson I do use sourcetree, but it seems to be jacked up on my new machine.

**Comment by LHinson** [comment]
thank you @jschuler!

File: packages/patternfly-4/content/design-guidelines/usage-and-behavior/lists-and-tables/design.md [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
-PatternFly offers two components for displaying large data sets: the data list and the data table. While they satisfy similar use cases, choosing the correct component to use in your design will be dependent on the type of data you need to display.
+PatternFly offers two components for displaying large data sets: [data list](https://www.patternfly.org/v4/documentation/react/components/datalist/) and [table](https://www.patternfly.org/v4/documentation/react/components/table/). While they satisfy similar use cases, choosing the correct component to use in your design will be dependent on the type of data you need to display.
 
 Use [data lists](#data-lists) when:
 * A flexible layout is more important than arranging information in a grid
 * You want to include active content like a chart
 * Content displayed may vary between rows
 
-Use a [data table](#data-tables) when:
+Use a [tables](#tables) when:
```
Comment (gdoyle1) [inline packages/patternfly-4/content/design-guidelines/usage-and-behavior/lists-and-tables/design.md]: This should be "Use [tables](#tables) when:" -- remove the "a"

**Comment by gdoyle1** [comment]
@ncameronbritt Only found one thing to fix here! Also we say "data table" here https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/toolbar#examples - could you include that fix in this PR as well?

**Comment by ncameronbritt** [comment]
Addressed comments. The [html demo data table](https://www.patternfly.org/v4/documentation/core/demos/datatable) still needs to be done. I'm happy to do it, but I would need some guidance--I can't find the file in my for of the patternfly-org repo.

**Comment by gdoyle1** [comment]
@ncameronbritt Thanks! I'll check with the team to see how we can change that demo. I'll create a separate issue though and just reference this one!

File: packages/patternfly-4/content/design-guidelines/usage-and-behavior/lists-and-tables/design.md [hunk 2]
```diff
@@ -2,14 +2,14 @@
 path: "/design-guidelines/usage-and-behavior/lists-and-tables"
 ---
 # Lists and tables
-PatternFly offers two components for displaying large data sets: the data list and the data table. While they satisfy similar use cases, choosing the correct component to use in your design will be dependent on the type of data you need to display.
+PatternFly offers two components for displaying large data sets: [data list](https://www.patternfly.org/v4/documentation/react/components/datalist/) and [table](https://www.patternfly.org/v4/documentation/react/components/table/). While they satisfy similar use cases, choosing the correct component to use in your design will be dependent on the type of data you need to display.
```
Comment (jschuler) [inline packages/patternfly-4/content/design-guidelines/usage-and-behavior/lists-and-tables/design.md]: we should use relative URLs here

Comment (ncameronbritt) [reply packages/patternfly-4/content/design-guidelines/usage-and-behavior/lists-and-tables/design.md]: Happy to make that change. Just tell me what to put for the url. :-)

**Comment by jschuler** [comment]
@ncameronbritt @gdoyle1 I modified the URLs to be relative. Does the PR look good to you?

**Comment by gdoyle1** [comment]
@jschuler this looks good - data table looks like it still appears in core demos although it has been changed on their workspace. Is there a way we can fix this as well? Or should I create a separate issue?

**Comment by jschuler** [comment]
@gdoyle1 yes a separate issue would be good. If not already done so, this should first be fixed upstream in @patternfly/patternfly-next repo, then when we can update the site to pull it in

**Comment by gdoyle1** [comment]
@jschuler Thanks! I believe it has been fixed in the patternfly-next repo already but will check. This is all set then

**Comment by ncameronbritt** [comment]
lgtm. Thanks @jschuler

**Pull request merged**